### DISCUSSION TITLE
Change Site-Editor `background-color`?

### DISCUSSION THREAD

**Discussion by r-i-c-h** [WordPress/gutenberg#57441] [Q&A]
It's impossible to see what my Patterns look like because of the dark background of the "Site-Editor" UI at /wp-admin/site-editor.php ([See screenshot](https://snipboard.io/PqQUCA.jpg ))

**How can I change the color of the area on the right hand side** to have a lighter background so I can actually preview anything?

I'll admit I might have somehow changed my Editor to a "Dark-Mode" but I don't remember doing that, and if I did, I sure can't find that setting.

**Comment by t-hamano** [comment]
Hi @r-i-c-h,

Could you tell me the following to find out the cause of this problem?

- What version of WordPress is it? Also, if you have the Gutenberg plugin enabled, what version is it?
- Does this problem also occur when Twenty Twenty-Four is enabled?

**Reply by r-i-c-h** [reply]
Using WP ver 6.4.2.  No Gutenberg plugin. *Is* OK when Twenty Twenty-Four is enabled.
Thank you for pointing me in the right direction. 

The editor uses `var(--wp--preset--color--base)` for generating the backgrounds on each preview widget.

The problem came from my Twenty Twenty-Four child-theme generated by the "Create Block Theme" plugin. 

I added a custom color `palette[]` in my child's `theme.json`, but I did _not_ include a color object with a value for the slug _`base`_!  

Added an object `{ "color": "#c0ffee", "name": "Base", "slug": "base" }` to the child `theme.json`'s `settings.color.palette[]` and fixed the issue. 

It is worrysome that the WP engine didn't attempt to grab the missing "`base`" value from Twenty Twenty-Four's `theme.json`.

In the child's `style.css` I do have the line `Template: twentytwentyfour`. Do I somehow have my child misconfigured or does the child's palette completely override the parent's palette?
